<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrator_widget</class>
 <widget class="QWidget" name="Administrator_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统管理员界面</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>580</width>
     <height>550</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>550</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>150</y>
       <width>92</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Cold Dish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Main Course</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Soup</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>41</y>
       <width>108</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
font: 75 20pt &quot;华文楷体&quot;;</string>
     </property>
     <property name="text">
      <string>菜单管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="Menu_tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>401</width>
       <height>411</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>350</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>编号（ID）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>菜名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>价格</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="showmenu_pushButton">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>81</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>显示菜单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addfood_pushButton">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>291</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>增加菜品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchfood_pushButton">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>221</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>查询菜品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletefood_pushButton">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>361</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>删除菜品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadfood_pushButton">
     <property name="geometry">
      <rect>
       <x>471</x>
       <y>431</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>上传数据库</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>436</width>
       <height>456</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
font: 75 20pt &quot;华文楷体&quot;;</string>
        </property>
        <property name="text">
         <string>账户管理</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>40</number>
        </property>
        <item>
         <widget class="QTableWidget" name="Account_tableWidget">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>350</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>用户类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>手机号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>密码</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>40</number>
          </property>
          <item>
           <widget class="QPushButton" name="showC_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>显示顾客账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showE_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>显示雇员账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="search_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>查询账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>增加账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>删除账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upload_pushButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>上传数据库</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>菜单管理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>账户管理</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/images/customer.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
